src/debug.c

```diff
@@ -713,7 +713,10 @@ NULL
         if (getPositiveLongFromObjectOrReply(c, c->argv[2], &keys, NULL) != C_OK)
             return;
 
-        dictExpand(c->db->dict,keys);
+        if (dictTryExpand(c->db->dict, keys) != DICT_OK) {
+            addReplyErrorFormat(c, "OOM in dictTryExpand %ld", keys);
```

Comment:
```suggestion
            addReplyError(c, "OOM in dictTryExpand");
```
You gave it the number of keys, would rather not just have some dangling part of the error.